package.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "inkjs",
-  "version": "2.2.4",
+  "version": "2.2.5",
   "description": "A javascript port of inkle's ink scripting language (http://www.inklestudios.com/ink/)",
   "type": "commonjs",
   "main": "dist/ink-full.js",
   "types": "ink.d.ts",
-  "files": ["src/engine","src/compiler","ink.d.ts", "ink.d.mts","script/inkjs-compiler.ts", "dist"],
+  "files": ["engine","compiler","ink.d.ts", "ink.d.mts","script/inkjs-compiler.ts", "dist"],
```

Comment:
Mistakenly exported the source files instead of the built files, sorry about that !